tests/integration/test_design.py

```diff
@@ -2010,6 +2011,50 @@ def test_set_fill_style(modeler: Modeler):
     assert box.fill_style == FillStyle.OPAQUE
 
 
+def test_set_body_color(modeler: Modeler):
+    """Test the getting and setting of body color."""
+    skip_if_linux(modeler, test_set_body_color.__name__, "set_color")  # Skip test on Linux
+
+    design = modeler.create_design("RVE2")
+    unit = DEFAULT_UNITS.LENGTH
+
+    plane = Plane(
+        Point3D([1 / 2, 1 / 2, 0.0], unit=unit),
+        UNITVECTOR3D_X,
+        UNITVECTOR3D_Y,
+    )
+    box_plane = Sketch(plane)
+    box_plane.box(Point2D([0.0, 0.0]), width=1 * unit, height=1 * unit)
+    box = design.extrude_sketch("Block", box_plane, 1 * unit)
+
+    # Default body color is if it is not set on server side.
+    assert box.color == "#000000"
```

Comment:
is the default color not that spaceclaim green?

Reply:
@jonahrb, SC GetColor returns null on default case, so its just defaulted to this APIServer.